<?xml version="1.0"?>
<launch>
  
  <node name="scan_26xx_node" pkg="scan_26xx" type="scan_26xx_node" cwd="node" respawn="true" output="screen"/>



</launch>
